src/main.rs

```diff
@@ -61,7 +59,23 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let host = IpAddr::from_str(&host_raw)?;
     let port = env::var("PORT").unwrap_or_else(|_| "80".into()).parse()?;
 
-    let client: Client<_, Body> = Client::builder().build(HttpsConnector::with_webpki_roots());
+    let https_connector = {
+        let mut http_connector = new_trust_dns_http_connector();
+        http_connector.enforce_http(false);
+
+        let builder = HttpsConnectorBuilder::new()
+            .with_webpki_roots()
+            .https_only()
+            .enable_http1();
+
+        if env::var("DISABLE_HTTP2").is_ok() {
```

Comment:
let's add some documentation about this, unless you consider it part of #45

Reply:
done for now, will probably extend this when I get to #45